src/main/java/uk/gov/cslearning/catalogue/api/MediaController.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.cslearning.catalogue.api;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.cslearning.catalogue.domain.media.Media;
+import uk.gov.cslearning.catalogue.service.FileUploadFactory;
+import uk.gov.cslearning.catalogue.service.MediaManagementService;
+
+@Controller
+@RequestMapping("/service/media")
```

Comment:
INFO - Is the endpoint mapped after my example or /service/ is a standard? if it was by my example, please just use `media`